etcdctl/README.md

```diff
@@ -736,9 +736,10 @@ If NOSPACE alarm is present:
 
 ### DEFRAG
```

Comment:
we need to mention the new option into the readme. example: https://github.com/coreos/etcd/tree/master/etcdctl#get-options-key-range_end